CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Added
 - Robots meta tags to control indexing of pages
+- WEB-229 cropping at bottom of hero
```

Comment:
this should be under `Fixed`

Reply:
But It's not a fix/bug it's a change due to the gradient being removed

Reply:
oh